app/index.tsx

```diff
@@ -0,0 +1,21 @@
+import Login from "./Login";
+import React from "react";
+import Loading from "./Loading";
+
+export default function App() {
+  const [isReady, setReady] = React.useState(false);
+
+  React.useEffect(() => {
+    // Perform some sort of async data or asset fetching.
+    setTimeout(() => {
+      setReady(true);
+    }, 2000);
+  }, []);

```

Comment:
Is this necessary? I think it can work without including this

Reply:
on it